cmp/form/FormModel.js

```diff
@@ -44,6 +43,9 @@ export class FormModel {
     /** @member {Object} - container object for FieldModel instances, keyed by field name. */
     @observable.ref fields = {};
 
+    /** @return {FieldModel[]} - all FieldModel instances, as an array. */
+    get allFields() {return values(this.fields)}
```

Comment:
Open of course to other suggestions on the name here, if you think the overall addition is worth it.  E.g. `fieldList`?